_data/people.yml

```
@@ -16,6 +16,15 @@
   office: WS403
   office-link: https://maps.app.goo.gl/oxxnz78nM16dp9eo8
   cosupervisors: Craig Butts & Varinder Aggarwal
+- name: Harry Richardson
+  role: PhD Student
+  email: bb24144@bristol.ac.uk
+  github: Harry-Rich
+  joined: Sept. 2024
+  homepage: https://www.linkedin.com/in/harry-richardson-834a30176/
+  office: W233
+  office-link: https://maps.app.goo.gl/oxxnz78nM16dp9eo8
+  cosupervisors: Andrew McCluskey & Neil Allan
```

I think for this, just have Neil. 
```suggestion
  cosupervisors: Neil Allan
```